bedrock/firefox/templates/firefox/new/trailhead/thanks-b.html

```diff
@@ -0,0 +1,107 @@
+{% add_lang_files "firefox/new/trailhead" %}
+
+{% extends "firefox/new/trailhead/base.html" %}
+
+{# "scene2" page should not be indexed to avoid it appearing in search results: issue 7024 #}
+{% block canonical_urls %}<meta name="robots" content="noindex,follow">{% endblock %}
+
+{% block extrahead %}
+  {{ super() }}
+  {{ css_bundle('firefox_new_thanks') }}
+{% endblock %}
+
+{% block string_data %}
+  data-pixels="{% for pixel in settings.TRACKING_PIXELS %}{{ pixel }}{% if not loop.last %}::{% endif %}{% endfor %}"
+{% endblock %}
+
+{% block site_header %}
+  {% with hide_nav_download_button=True %}
+    {% include 'includes/protocol/navigation/menu-firefox/index.html' %}
+  {% endwith %}
+{% endblock %}
+
+{% block content %}
+<main role="main">
+  <div class="mzp-l-content">
+    <div class="mzp-c-notification-bar mzp-t-warning unsupported">
+      <p>
+        {% trans url='https://support.mozilla.org/kb/end-support-windows-xp-and-vista' %}
+        You’re using an insecure, outdated operating system <a href="{{ url }}">no longer supported by Firefox</a>.
+        {% endtrans %}
+      </p>
+    </div>
+    <div class="mzp-c-notification-bar mzp-t-success auto-download">
+      {# fallback_url is replaced by the platform download link via JS, but if
+      something fails the user should still get a link to a working download path. #}
+      <p>
+        {% trans id='direct-download-link', fallback_url=url('firefox.all') %}
+        Your download should begin automatically. Didn’t work? <a id="{{ id }}" href="{{ fallback_url }}">Try
+          downloading again</a>.
+        {% endtrans %}
+      </p>
+    </div>
+  </div>
+
+  {% call call_out(
+  title=_('You’re almost done…'),
+  heading_level=1,
+  include_cta=True
+  ) %}
+  <div id="download-button-wrapper-desktop">
+    {{ download_firefox(force_direct=true, dom_id='primary-download-button') }}
+  </div>
+  <h2>{{ _('Install Firefox in 2 easy steps') }}</h2>
+  {% endcall %}
+
+  <section class="mzp-l-content">
+    <ol class="l-columns-two">
+      <li class="c-picto-block">
+        <h3 class="c-picto-block-title">{{ _('Click on the Firefox installer.') }}</h3>
+        <div class="c-picto-block-image">
+          <img src="{{ static('img/firefox/new/trailhead/browser-status-bar.png') }}" alt="">
+        </div>
+        <div class="c-picto-block-body">
+          <p><strong>{{ _('Look down, look left.') }}</strong> {{ _('Click on the Firefox installer in the lower bottom
+            left-hand corner.') }}</p>
+        </div>
+      </li>
+
+      <li class="c-picto-block">
+        <h3 class="c-picto-block-title">{{ _('Select ‘Yes.’') }}</h3>
+        <div class="c-picto-block-image">
+          <img src="{{ static('img/firefox/new/trailhead/browser-modal.png') }}" alt="">
+        </div>
+        <div class="c-picto-block-body">
+          <p>
+            {% trans %}
+            Click YES for Firefox. <strong>Select YES</strong> on the pop-up window to finish the install.
```

Comment:
This is true for Windows 10, but do we need to worry about other Windows versions in this experiment?

I also wonder if this is going to pose a challenge for L10n, since translators may not have access to a Windows version in their language to test?

Reply:
> This is true for Windows 10, but do we need to worry about other Windows versions in this experiment?

We are going to target Windows 10 specifically. I am going to work on creating the audience in convert based on user agent. :+1:

> I also wonder if this is going to pose a challenge for L10n, since translators may not have access to a Windows version in their language to test?

I am not sure what the ultimate localization plan for this one would end up being, but you are right it would need a larger conversation.